circle.yml

```diff
@@ -16,16 +16,14 @@ dependencies:
     - pip uninstall -y docker || true
     - pip install docker || true
     - pip install codecov
-    - ./gradlew classes testClasses
-
-  cache_directories:
-    - ~/.gradle
+    - mkdir -p ~/.gradle && echo "org.gradle.daemon=false" >> ~/.gradle/gradle.properties
```

Comment:
Gradle daemon seemed to be running by default and breaking the functional tests by causing them to hang.